services/QuillLMS/app/models/activity_session_feedback_history.rb

```diff
@@ -0,0 +1,44 @@
+# == Schema Information
+#
+# Table name: activity_session_feedback_histories
+#
+#  id                           :integer          not null, primary key
+#  activity_session_uid         :string
+#  feedback_history_session_uid :string
+#
+# Indexes
+#
+#  index_activity_sess_fb_histories_on_activity_session_uid  (activity_session_uid) UNIQUE
+#  index_activity_sess_fb_histories_on_feedback_history_uid  (feedback_history_session_uid) UNIQUE
+#
+class ActivitySessionFeedbackHistory < ActiveRecord::Base
```

Comment:
I think we should change this name to something like `FeedbackSession`. I think you were trying to name this as a Rails join table, but it's joining on uids and is creating an new entity/concept of the feedback session that goes beyond the basic join table.

Also, I think this will be clearer longterm if this `feedback_session.uid` vs `activity_session_feedback_history.feedback_history_session_uid`